drag_and_drop_v2/compat.py

```diff
@@ -0,0 +1,29 @@
+"""
+Compatibility layer to isolate core-platform waffle flags from implementation.
+"""
+
+# Waffle flags configuration
+
+# Namespace
+WAFFLE_NAMESPACE = "drag_and_drop_v2"
+
+# Course Waffle Flags
+# .. toggle_name: drag_and_drop_v2.enable_alternative_grading
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Enables alternative grading for the xblock
+#    that does not include decoy items in the score.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2022-11-10
+# .. toggle_tickets: <>
+# .. toggle_warning: None.
+ENABLE_ALTERNATIVE_GRADING = 'enable_alternative_grading'
+
+
+def get_config_waffle_flag():
```

Comment:
Nit: this one does not have anything to do with the "config". We could call it, e.g. `get_grading_ignore_decoys_waffle_flag`.